references/identity-facade/pipeline.sh

```diff
@@ -0,0 +1,356 @@
+                        },
+                        {
+                            "name": "idp.token.hostname",
+                            "value": "$TEST_IDP_TOKEN_HOSTNAME"
+                        },
+                        {
+                            "name": "idp.token.uri",
+                            "value": "$TEST_IDP_TOKEN_URI"
+                        },
+                        {
+                            "name": "idp.jwks.hostname",
+                            "value": "$TEST_IDP_JWKS_HOSTNAME"
+                        },
+                        {
+                            "name": "idp.jwks.uri",
+                            "value": "$TEST_IDP_JWKS_URI"
+                        },
+                        {
+                            "name":"idp.userinfo.hostname",
+                            "value":"$TEST_IDP_USERINFO_HOSTNAME"
+                        },
+                        {
+                            "name":"idp.userinfo.uri",
+                            "value":"$TEST_IDP_USERINFO_URI"
+                        },
+                        {
+                            "name":"idp.issuer",
+                            "value":"$TEST_IDP_ISSUER"
+                        }
+                    ]
+                }
+            ],
+	    "extensions":[ ]
+        }
+    },
+    "orgConfig": {
+        "apiProducts": [
+            {
+                "name": "IdentityFacade",
+                "apiResources": [],
+                "approvalType": "auto",
+                "attributes": [
+                    {
+                        "name": "access",
+                        "value": "public"
+                    }
+                ],
+                "description": "",
+                "displayName": "Identity Facade",
+                "environments": [
+                    "test"
+                ],
+                "proxies": [
+                    "identity-facade-v1"
+                ]
+            }
+        ],
+        "developers": [
+            {
+                "email": "janedoe@example.com",
+                "firstName": "Jane",
+                "lastName": "Doe",
+                "userName": "janedoe",
+                "attributes": []
+            }
+        ],
+        "developerApps": {
+            "janedoe@example.com": [
+                {
+                    "name": "identityApp",
+		    "attributes":[
+			{
+			  "name":"displayName",
+			  "value":"identity App"
+			}
+		    ],
+                    "apiProducts": [ "IdentityFacade" ],
+                    "callbackUrl": "https://httpbin.org/get",
+                    "scopes": []
+                }
+            ]
+        }
+    },
+    "apiConfig": {}
+}
+EOF
+}
+
+####################################################
+### function: generate_post_data_app_credentials ###
+####################################################
+generate_post_data_app_credentials()
+{
+  cat <<EOF
+{
+  "consumerKey": "xkey",
```

Comment:
these must be unique _per planet_ (which is a bit strange)

so we could add a timestamp or something to make sure they are unique